DESCRIPTION

```diff
@@ -123,7 +125,8 @@ Suggests:
     testthat (>= 3.0.0)
 Remotes:
     Bioconductor/BiocParallel,
-    mixOmicsTeam/mixOmics
+    mixOmicsTeam/mixOmics,
+    simonpcouch/forested
```

Comment:
I _think_ the CRAN version is up to date! We don't plan on switching out the data to different states in that package, for now.